### ISSUE TITLE
Feature Request Debug/Alternative Stream

### ISSUE THREAD

**Issue by 1fxe** [pystardust/ani-cli#196]
Sometimes stream return invalid(?) urls? and causes a HTTP Error 404, if I check in my browser with GoGoServer it loads fine.
There is no log in the selection menu indicating it failed to load the show. 

So my request is to either pint to the user no stream found. Or parse an alternative stream

```
[   0.220][d][ytdl_hook] Running: yt-dlp --no-warnings -J --flat-playlist --sub-format ass/srt/best --format bestvideo+bestaudio/best --all-subs --no-playlist -- https://hlsx02.asiancdn.net/astream/c751bab1939a2b83020565e1ac242896/one-piece-episode-1.1637610692..m3u8 
[   0.220][d][cplayer] Run command: subprocess, flags=64, args=[args="yt-dlp,--no-warnings,-J,--flat-playlist,--sub-format,ass/srt/best,--format,bestvideo+bestaudio/best,--all-subs,--no-playlist,--,https://hlsx02.asiancdn.net/astream/c751bab1939a2b83020565e1ac242896/one-piece-episode-1.1637610692..m3u8", playback_only="yes", capture_size="67108864", capture_stdout="yes", capture_stderr="yes", detach="no", env="", stdin_data="", passthrough_stdin="no"]
[   1.041][v][ytdl_hook] Found youtube-dl with path yt-dlp in PATH 
[   1.041][e][ytdl_hook] ERROR: [generic] Unable to download webpage: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. 
[   1.041][e][ytdl_hook] youtube-dl failed: unexpected error occurred 
[   1.041][v][cplayer] finished playback, loading failed (reason 4)
```

**Comment by TheGlockMisc** [comment]
Problem on Parsing Specific Quality
Same issue in #192 
> There must be a specific quality among those 2 dots `.1080.m3u8 `

**Comment by ura43** [comment]
Yep, closing in favor of #192 which was closed in favor of #202
